packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -213,7 +212,7 @@ beforeAll( async () => {
 
 	await trashExistingPosts();
 	await setupBrowser();
-	await activatePlugin( 'gutenberg-test-plugin-disables-the-css-animations' );
+	await page.emulateMediaFeatures( [ { name: 'prefers-reduced-motion', value: 'reduce' } ] );
```

Comment:
How nice 🎉

Reply:
Still waiting for the confirmation from Travis :P

Reply:
I wonder if we should move it to each page load.

Reply:
Right, this might be required for every page load 🙃 It doesn't work at the moment according to Travis.